bin/plot_truncated_pexecs

```diff
@@ -190,19 +190,17 @@ def generate_trials(json_file):
     summary = dict()
     for n_pexecs in xrange(MIN_PEXECS, MAX_PEXECS + 1):
         summary[n_pexecs] = list()
-        # for trial in xrange(NUMBER_TRIALS):
-        #     summary[n_pexecs][.append(dict())]
-
-    # for n_pexecs in data_copies:
-        # summary[n_pexecs] = list()
-        # for trial in xrange(NUMBER_TRIALS):
-        #     summary[n_pexecs].append(dict())
-        #     for key in data_copies[n_pexecs][trial]:
-        #         summary[n_pexecs][trial] = collect_summary_statistics({machine: data_copies[n_pexecs][trial]}, delta, steady)
-
-    print 'Start parallel'
-    # FIXME: NEEDS CHANGING BY HAND
-    start_ends = [(2, 5), (5, 10), (10, 15)]  # [inclusive, exclusive) intervals.
+
+    print 'Start parallel execution.'
+    # start_ends describes which work is done on which core, and must match the
+    # physical machine. Intervals are [inclusive, exclusive).
+    start_ends = [(2, 11), (10, 21), (20, 31)]  # 4-core machine.
+    # start_ends = [(2, 3), (3, 4), (4, 5), (5, 6),  # 32-core machine.
```

Comment:
I'm not sure I'm super comfortable with this in the PR, even though I know why it's there. It's such a hack that I feel we should either maintain it locally or make it some sort of feature (e.g. we could provide settings for 4, 8, 16, and 32 core machines).